src/guide/jbs-jdk-bug-system.md

```diff
@@ -5,52 +5,300 @@
+
+## Issue states
+
+JBS only has a few states in which an issue can be in:
+
+<table class="dictionary" summary="JBS Label Dictionary">
+  <tr style="text-align:left;"><th>Type</th><th>Covers</th></tr>
+<tr>
+    <td class="dictionary">[New]{.jbs-field}</td>
+    <td class="dictionary">Initial state after an issue is filed. [Bugs]{.jbs-field} in the JDK Project must be taken out of the [New]{.jbs-field} state (Triaged - see below) in a timely manner. In general, this is recommended to be done for all issue types and projects as a sign that the issue is correctly filed, and will be seen by the right group - this is especially important towards the end of a release.</td>
+</tr>
+<tr>
+    <td class="dictionary">[Open]{.jbs-field}</td>
+    <td class="dictionary">Once the issue has been triaged it will be in the [Open]{.jbs-field} state and will stay here until the assignee decides to work on it, at which point it is encouraged that the "Start Work" option be selected to move it to [In Progress]{.jbs-field}.</td>
+</tr>
+<tr>
+    <td class="dictionary">[In Progress]{.jbs-field}</td>
+    <td class="dictionary">The [In Progress]{.jbs-field} state has the option of the sub-states ([Understanding]{.jbs-field}): [Cause Known]{.jbs-field}, [Fix Understood]{.jbs-field}, [In Review]{.jbs-field}.</td>
+</tr>
+<tr>
+    <td class="dictionary">[Closed]{.jbs-field}/[Resolved]{.jbs-field}</td>
+    <td class="dictionary">While [Closed]{.jbs-field} and [Resolved]{.jbs-field} are essentially equivalent, when it comes to fixing issues there is an additional [Verify]{.jbs-field} step available between the [Resolved]{.jbs-field} and [Closed]{.jbs-field} states.</td>
+</tr>
+</table>
+
+::: {style="text-align:center;"}
+~~~{.mermaid caption="JBS Issue Flow" format=svg theme=neutral}
+flowchart TD
+
+subgraph top[" "]
+
+New
+triage
+ResolvedIncomplete
+ClosedIncomplete
+Open
+InProgress
+states
+Resolved
+Verify
+Closed
+
+  New(NEW) --> triage
+  Open --> |Move to new component|New
+
+  triage --> |sync with submitter|ResolvedIncomplete(<font color=white>RESOLVED:Incomplete)
+  ResolvedIncomplete --> |Clarified|triage
+  ResolvedIncomplete --> |Not Clarified|ClosedIncomplete
+  ClosedIncomplete(<font color=green>CLOSED:Incomplete) --> Closed
+
+  triage --> Open(OPEN)
+  Open --> states
+
+  Open <--> InProgress(<font color=blue>IN PROGRESS)
+  InProgress --> states
+
+  states[Fixed, Won't Fix, Duplicate etc.] --> Closed
+  states --> Resolved
+  Resolved(<font color=white>RESOLVED) -.-> Verify[Verify]
+  Verify --> Closed(<font color=green>CLOSED)
+end
+
+style top fill:white
+style New fill:lightgrey
+style Open fill:lightgrey
+style triage fill:white
+style states fill:white
+style Verify fill:white
+style Resolved fill:green
+style ResolvedIncomplete fill:green
+style Closed fill:lightgreen
+style ClosedIncomplete fill:lightgreen
+~~~
+:::
+
+## Triaging an issue
```

Comment:
Correct see line 184:
If the issue belongs to a different area (it was filed in libraries, but it is an HotSpot issue), transfer it to the correct component/subcomponent making sure that the state remains New.

Reply:
Right. The difference is that this situation happens *after* the initial triage, which means that the state has already been moved to Open. It's important that the engineer, which often are not part of the triage team, move the bug back to the New state. Maybe it's not worth spelling out this situation explicitly when it can be implied from line 184.